### DISCUSSION TITLE
school email name does not match my school

### DISCUSSION THREAD

**Discussion by circularsprojects** [community/community#54713] [GitHub Education]
### Hi everyone,

I’m struggling with reverifying my student developer lack because github keeps thinking that my school name is “NSW Department of Education”
(that’s not even a school, that’s the name of the government department behind public schooling in australia)
I’ve tried to submit my student ID, but got rejected because my school name wasn’t in the photo. (it was)
I don’t know what else i’m meant to do because i cant just change the name of my school when submitting the form to be the actual name of my school and not “department of education”

any help or anyone facing the same issue?

### A suggestion...(optional)

support

**Comment by samus-aran** [comment]
Hi @circularsprojects 👋 

Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. 

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "Other". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and I'll be happy to assist!
